src/jobservice/job/impl/logger/job_logger.go

```diff
@@ -15,35 +15,78 @@
 package logger
 
 import (
+	"errors"
+	"fmt"
 	"os"
+	"path"
 	"strings"
 
-	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/jobservice/config"
 	"github.com/goharbor/harbor/src/jobservice/logger"
+	logging "github.com/op/go-logging"
 )
 
+var format = logging.MustStringFormatter(
+	`%{color}%{time:15:04:05.000} %{shortfunc} ▶ %{level:.4s} %{id:03x}%{color:reset} %{message}`,
+)
+var moduleName = "JobService"
+
 // JobLogger is an implementation of logger.Interface.
 // It used in the job to output logs to the logfile.
 type JobLogger struct {
-	backendLogger *log.Logger
+	backendLogger *logging.Logger
 	streamRef     *os.File
 }
 
 // New logger
-// nil might be returned
-func New(logPath string, level string) logger.Interface {
-	f, err := os.OpenFile(logPath, os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		logger.Errorf("Failed to create job logger: %s", err)
-		return nil
+func New(jobID string) (logger.Interface, error) {
```

Comment:
The way the job logger is configured, makes it seems there should be multiple implementation of the interface, rather than switch to go-logging and add different backends to it.